config/crd/bases/eventing.keda.sh_cloudeventsources.yaml

```diff
@@ -44,11 +44,32 @@ spec:
           spec:
             description: CloudEventSourceSpec defines the spec of CloudEventSource
             properties:
+              authenticationRef:
+                description: |-
+                  AuthenticationRef points to the TriggerAuthentication or ClusterTriggerAuthentication object that
+                  is used to authenticate the scaler with the environment
+                properties:
+                  kind:
+                    description: Kind of the resource being referred to. Defaults
+                      to TriggerAuthentication.
+                    type: string
+                  name:
+                    type: string
+                required:
+                - name
+                type: object
               clusterName:
                 type: string
               destination:
                 description: Destination defines the various ways to emit events
                 properties:
+                  azureEventGridTopic:
+                    properties:
+                      endPoint:
```

Comment:
```suggestion
                      endpoint:
```

Reply:
This code (all the CRDs indeed) is autogenerated, so manual changes aren't worth xD

Reply:
I know, just making sure all is fine when I do final review. Also posted on https://github.com/kedacore/keda/pull/5606#discussion_r1544267867